offixsdk/src/main/java/org/aerogear/offixsdk/worker/OfflineMutationsWorker.kt

```diff
@@ -0,0 +1,115 @@
+package org.aerogear.offixsdk.worker
+
+import android.content.Context
+import android.util.Log
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import com.apollographql.apollo.ApolloCall
+import com.apollographql.apollo.api.Input
+import com.apollographql.apollo.api.Mutation
+import com.apollographql.apollo.api.Operation
+import com.apollographql.apollo.api.Response
+import com.apollographql.apollo.exception.ApolloException
+import org.aerogear.offixsdk.Application
+import org.aerogear.offixsdk.ClientGenerator
+
+
+/**
+ * Workmanager that replicates the mutations to the server that are stored in the database.
+ * Post firing the mutation, it is deleted from the db
+ */
+
+class OfflineMutationsWorker(context: Context, workerParameters: WorkerParameters) : Worker(context, workerParameters) {
+
+
+    private val getMutationDao = Application.database.mutationDao()
+    private val getUrlDao = Application.database.urlDao()
+
+    val TAG = javaClass.simpleName
```

Comment:
Maybe create an extension function for Log in which you take `javaClass.simpleName` automatically so that the developer only has to write the log message and not the TAG again and again